2020-08-13 00:22:38.915 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 00:22:38.957 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 00:22:39.051 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 00:22:39.079 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 00:22:39.107 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 00:22:39.208 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 00:22:39.234 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 00:22:39.748 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 00:22:39.748 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 00:22:39.810 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 00:22:39.836 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 00:22:39.853 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 00:22:39.927 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 00:22:39.957 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 00:22:39.999 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 00:22:40.001 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 00:22:40.060 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 00:22:40.092 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 00:22:40.162 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 00:22:40.252 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 00:22:40.300 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 00:22:40.310 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 00:22:40.323 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 00:22:40.372 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 00:22:40.419 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 00:22:40.434 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 00:22:40.476 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 00:22:40.654 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 00:22:40.679 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 00:22:40.742 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 00:22:40.766 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 00:22:43.302 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 00:22:44.370 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.580.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.580.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:22:44.410 [error] <0.580.0> CRASH REPORT Process <0.580.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:22:44.410 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.580.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:22:47.157 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.849.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.849.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:22:47.157 [error] <0.849.0> CRASH REPORT Process <0.849.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:22:47.157 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.849.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:22:49.818 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.854.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.854.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:22:49.818 [error] <0.854.0> CRASH REPORT Process <0.854.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:22:49.818 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.854.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:22:52.144 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.855.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.855.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:22:52.145 [error] <0.855.0> CRASH REPORT Process <0.855.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:22:52.145 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.855.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:22:54.519 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.856.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.856.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:22:54.520 [error] <0.856.0> CRASH REPORT Process <0.856.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:22:54.520 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.856.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:22:56.909 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.857.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.857.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:22:56.909 [error] <0.857.0> CRASH REPORT Process <0.857.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:22:56.910 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.857.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:22:59.238 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.858.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.858.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:22:59.238 [error] <0.858.0> CRASH REPORT Process <0.858.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:22:59.238 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.858.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:01.592 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.859.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.859.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:01.592 [error] <0.859.0> CRASH REPORT Process <0.859.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:01.593 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.859.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:04.159 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.860.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.860.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:04.159 [error] <0.860.0> CRASH REPORT Process <0.860.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:04.160 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.860.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:06.601 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.861.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.861.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:06.601 [error] <0.861.0> CRASH REPORT Process <0.861.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:06.603 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.861.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:09.015 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.862.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.862.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:09.015 [error] <0.862.0> CRASH REPORT Process <0.862.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:09.015 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.862.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:11.366 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.863.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.863.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:11.366 [error] <0.863.0> CRASH REPORT Process <0.863.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:11.366 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.863.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:13.718 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.864.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.864.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:13.719 [error] <0.864.0> CRASH REPORT Process <0.864.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:13.719 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.864.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:16.070 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.865.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.865.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:16.070 [error] <0.865.0> CRASH REPORT Process <0.865.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:16.072 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.865.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:18.404 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.866.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.866.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:18.404 [error] <0.866.0> CRASH REPORT Process <0.866.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:18.405 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.866.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:20.798 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.867.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.867.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:20.800 [error] <0.867.0> CRASH REPORT Process <0.867.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:20.800 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.867.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:23.130 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.868.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.868.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:23.131 [error] <0.868.0> CRASH REPORT Process <0.868.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:23.131 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.868.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:25.443 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.869.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.869.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:25.444 [error] <0.869.0> CRASH REPORT Process <0.869.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:25.444 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.869.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:27.750 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.870.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.870.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:27.751 [error] <0.870.0> CRASH REPORT Process <0.870.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:27.751 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.870.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:30.090 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.871.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.871.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:30.090 [error] <0.871.0> CRASH REPORT Process <0.871.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:30.091 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.871.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:32.457 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.872.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.872.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:32.458 [error] <0.872.0> CRASH REPORT Process <0.872.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:32.458 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.872.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:34.776 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.873.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.873.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:34.777 [error] <0.873.0> CRASH REPORT Process <0.873.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:34.777 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.873.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:37.077 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.874.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.874.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:37.078 [error] <0.874.0> CRASH REPORT Process <0.874.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:37.078 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.874.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:39.392 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.875.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.875.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:39.392 [error] <0.875.0> CRASH REPORT Process <0.875.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:39.395 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.875.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:43.515 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.876.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.876.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:43.516 [error] <0.876.0> CRASH REPORT Process <0.876.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:43.516 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.876.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:45.835 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.877.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.877.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:45.836 [error] <0.877.0> CRASH REPORT Process <0.877.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:45.836 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.877.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:48.180 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.878.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.878.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:48.181 [error] <0.878.0> CRASH REPORT Process <0.878.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:48.181 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.878.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:50.589 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.879.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.879.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:50.590 [error] <0.879.0> CRASH REPORT Process <0.879.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:50.590 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.879.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:52.892 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.880.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.880.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:52.893 [error] <0.880.0> CRASH REPORT Process <0.880.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:52.893 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.880.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:55.271 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.881.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.881.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:55.272 [error] <0.881.0> CRASH REPORT Process <0.881.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:55.272 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.881.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:57.574 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.882.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.882.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:57.575 [error] <0.882.0> CRASH REPORT Process <0.882.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:57.575 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.882.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:23:59.878 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.883.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.883.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:23:59.879 [error] <0.883.0> CRASH REPORT Process <0.883.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:23:59.879 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.883.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:02.283 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.884.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.884.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:02.283 [error] <0.884.0> CRASH REPORT Process <0.884.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:02.284 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.884.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:04.614 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.885.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.885.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:04.615 [error] <0.885.0> CRASH REPORT Process <0.885.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:04.615 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.885.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:06.924 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.886.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.886.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:06.925 [error] <0.886.0> CRASH REPORT Process <0.886.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:06.925 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.886.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:09.283 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.887.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.887.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:09.283 [error] <0.887.0> CRASH REPORT Process <0.887.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:09.284 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.887.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:11.601 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.888.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.888.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:11.604 [error] <0.888.0> CRASH REPORT Process <0.888.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:11.604 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.888.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:14.042 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.889.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.889.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:14.042 [error] <0.889.0> CRASH REPORT Process <0.889.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:14.042 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.889.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:16.366 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.890.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.890.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:16.367 [error] <0.890.0> CRASH REPORT Process <0.890.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:16.367 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.890.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:18.683 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.891.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.891.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:18.683 [error] <0.891.0> CRASH REPORT Process <0.891.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:18.684 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.891.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:21.048 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.892.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.892.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:21.049 [error] <0.892.0> CRASH REPORT Process <0.892.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:21.049 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.892.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:24:23.341 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.893.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.893.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:24:23.342 [error] <0.893.0> CRASH REPORT Process <0.893.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:24:23.342 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.134983772.519307265.14940> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.893.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:25:57.706 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 00:25:57.728 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 00:25:57.791 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 00:25:57.820 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 00:25:57.841 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 00:25:57.894 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 00:25:57.916 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 00:25:58.161 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 00:25:58.161 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 00:25:58.183 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 00:25:58.189 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 00:25:58.207 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 00:25:58.239 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 00:25:58.261 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 00:25:58.307 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 00:25:58.307 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 00:25:58.351 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 00:25:58.379 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 00:25:58.427 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 00:25:58.476 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 00:25:58.510 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 00:25:58.519 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 00:25:58.528 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 00:25:58.577 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 00:25:58.592 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 00:25:58.599 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 00:25:58.631 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 00:25:58.718 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 00:25:58.734 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 00:25:58.746 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 00:25:58.763 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 00:26:00.991 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 00:26:02.083 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.580.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.580.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:26:02.094 [error] <0.580.0> CRASH REPORT Process <0.580.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:26:02.094 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.166990432.3740008451.240394> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.580.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:26:04.473 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.854.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.854.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:26:04.473 [error] <0.854.0> CRASH REPORT Process <0.854.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:26:04.473 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.166990432.3740008451.240394> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.854.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:29:41.866 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 00:29:41.890 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 00:29:41.946 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 00:29:41.972 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 00:29:41.988 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 00:29:42.017 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 00:29:42.038 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 00:29:42.264 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 00:29:42.264 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 00:29:42.282 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 00:29:42.288 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 00:29:42.308 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 00:29:42.345 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 00:29:42.371 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 00:29:42.428 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 00:29:42.428 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 00:29:42.474 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 00:29:42.496 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 00:29:42.542 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 00:29:42.583 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 00:29:42.612 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 00:29:42.620 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 00:29:42.629 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 00:29:42.675 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 00:29:42.683 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 00:29:42.690 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 00:29:42.734 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 00:29:42.807 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 00:29:42.822 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 00:29:42.845 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 00:29:42.860 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 00:29:44.379 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 00:29:46.023 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.580.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.580.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:29:46.036 [error] <0.580.0> CRASH REPORT Process <0.580.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:29:46.037 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.580.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:29:48.368 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.849.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.849.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:29:48.368 [error] <0.849.0> CRASH REPORT Process <0.849.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:29:48.368 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.849.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:29:50.670 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.850.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.850.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:29:50.670 [error] <0.850.0> CRASH REPORT Process <0.850.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:29:50.670 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.850.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:29:52.992 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.851.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.851.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:29:52.993 [error] <0.851.0> CRASH REPORT Process <0.851.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:29:52.993 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.851.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:29:56.321 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.852.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.852.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:29:56.323 [error] <0.852.0> CRASH REPORT Process <0.852.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:29:56.323 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.852.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:29:58.860 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.853.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.853.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:29:58.861 [error] <0.853.0> CRASH REPORT Process <0.853.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:29:58.862 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.853.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:02.663 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.857.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.857.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:02.664 [error] <0.857.0> CRASH REPORT Process <0.857.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:02.665 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.857.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:04.979 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.864.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.864.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:04.979 [error] <0.864.0> CRASH REPORT Process <0.864.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:04.980 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.864.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:07.283 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.865.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.865.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:07.283 [error] <0.865.0> CRASH REPORT Process <0.865.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:07.284 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.865.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:09.617 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.866.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.866.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:09.617 [error] <0.866.0> CRASH REPORT Process <0.866.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:09.617 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.866.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:11.947 [warning] <0.294.0> Unexpected error_logger event {error_msg,<0.439.0>,{<0.867.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.867.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:11.963 [error] <0.867.0> CRASH REPORT Process <0.867.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:11.975 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.2534029094.3740794881.123769> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.867.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:19.302 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 00:30:19.329 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 00:30:19.385 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 00:30:19.407 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 00:30:19.425 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 00:30:19.469 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 00:30:19.493 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 00:30:19.722 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 00:30:19.722 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 00:30:19.737 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 00:30:19.742 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 00:30:19.755 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 00:30:19.787 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 00:30:19.811 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 00:30:19.843 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 00:30:19.844 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 00:30:19.881 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 00:30:19.897 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 00:30:19.930 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 00:30:20.012 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 00:30:20.047 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 00:30:20.060 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 00:30:20.077 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 00:30:20.150 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 00:30:20.159 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 00:30:20.165 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 00:30:20.209 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 00:30:20.301 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 00:30:20.361 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 00:30:20.370 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 00:30:20.375 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 00:30:21.972 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 00:30:23.522 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.570.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.570.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:23.535 [error] <0.570.0> CRASH REPORT Process <0.570.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:23.536 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.1489486449.788529154.119384> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.570.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:25.897 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.842.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.842.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:25.897 [error] <0.842.0> CRASH REPORT Process <0.842.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:25.897 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.1489486449.788529154.119384> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.842.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:28.218 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.843.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.843.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:28.218 [error] <0.843.0> CRASH REPORT Process <0.843.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:28.218 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.1489486449.788529154.119384> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.843.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:30:30.523 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.844.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.844.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:30:30.523 [error] <0.844.0> CRASH REPORT Process <0.844.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:30:30.523 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.1489486449.788529154.119384> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.844.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:34:53.000 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 00:34:53.028 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 00:34:53.098 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 00:34:53.140 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 00:34:53.164 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 00:34:53.222 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 00:34:53.240 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 00:34:53.464 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 00:34:53.464 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 00:34:53.478 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 00:34:53.483 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 00:34:53.496 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 00:34:53.528 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 00:34:53.548 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 00:34:53.578 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 00:34:53.580 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 00:34:53.630 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 00:34:53.673 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 00:34:53.716 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 00:34:53.745 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 00:34:53.792 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 00:34:53.795 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 00:34:53.805 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 00:34:53.852 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 00:34:53.863 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 00:34:53.868 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 00:34:53.933 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 00:34:54.016 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 00:34:54.031 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 00:34:54.046 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 00:34:54.062 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 00:34:55.676 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 00:34:58.861 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.570.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.570.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:34:58.874 [error] <0.570.0> CRASH REPORT Process <0.570.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:34:58.874 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.388117818.1058013185.184087> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.570.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:35:02.364 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.841.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.841.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:35:02.364 [error] <0.841.0> CRASH REPORT Process <0.841.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:35:02.364 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.388117818.1058013185.184087> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.841.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:35:05.806 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.842.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.842.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:35:05.807 [error] <0.842.0> CRASH REPORT Process <0.842.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:35:05.807 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.388117818.1058013185.184087> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.842.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:35:09.087 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.843.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.843.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:35:09.088 [error] <0.843.0> CRASH REPORT Process <0.843.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:35:09.088 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.388117818.1058013185.184087> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.843.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:35:12.320 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.844.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.844.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:35:12.321 [error] <0.844.0> CRASH REPORT Process <0.844.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:35:12.321 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.388117818.1058013185.184087> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.844.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:37:23.477 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 00:37:23.507 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 00:37:23.579 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 00:37:23.608 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 00:37:23.627 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 00:37:23.674 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 00:37:23.698 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 00:37:23.924 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 00:37:23.925 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 00:37:23.941 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 00:37:23.945 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 00:37:23.962 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 00:37:23.991 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 00:37:24.015 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 00:37:24.045 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 00:37:24.045 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 00:37:24.077 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 00:37:24.092 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 00:37:24.146 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 00:37:24.179 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 00:37:24.228 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 00:37:24.230 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 00:37:24.244 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 00:37:24.293 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 00:37:24.305 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 00:37:24.309 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 00:37:24.346 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 00:37:24.429 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 00:37:24.481 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 00:37:24.499 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 00:37:24.510 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 00:37:25.871 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 00:37:28.708 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.570.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.570.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:37:28.724 [error] <0.570.0> CRASH REPORT Process <0.570.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:37:28.724 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.1308676475.2937061377.95687> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.570.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:45:53.902 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 00:45:53.927 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 00:45:54.007 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 00:45:54.046 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 00:45:54.067 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 00:45:54.116 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 00:45:54.138 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 00:45:54.370 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 00:45:54.370 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 00:45:54.383 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 00:45:54.387 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 00:45:54.400 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 00:45:54.490 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 00:45:54.510 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 00:45:54.538 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 00:45:54.538 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 00:45:54.566 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 00:45:54.586 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 00:45:54.614 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 00:45:54.648 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 00:45:54.683 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 00:45:54.697 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 00:45:54.713 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 00:45:54.762 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 00:45:54.770 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 00:45:54.777 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 00:45:54.812 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 00:45:54.883 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 00:45:54.902 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 00:45:54.910 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 00:45:54.915 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 00:45:56.605 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 00:45:59.419 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.570.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.570.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:45:59.433 [error] <0.570.0> CRASH REPORT Process <0.570.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:45:59.434 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.3515569155.2669936641.131262> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.570.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:46:02.820 [warning] <0.284.0> Unexpected error_logger event {error_msg,<0.429.0>,{<0.837.0>,[42,42,32,84,97,115,107,32,126,112,32,116,101,114,109,105,110,97,116,105,110,103,126,110,42,42,32,83,116,97,114,116,101,100,32,102,114,111,109,32,126,112,126,110,42,42,32,87,104,101,110,32,102,117,110,99,116,105,111,110,32,32,61,61,32,126,112,126,110,42,42,32,32,32,32,32,32,97,114,103,117,109,101,110,116,115,32,61,61,32,126,112,126,110,42,42,32,82,101,97,115,111,110,32,102,111,114,32,116,101,114,109,105,110,97,116,105,111,110,32,61,61,32,126,110,42,42,32,126,112,126,110],[<0.837.0>,'Elixir.PuzzlespaceWeb.Endpoint',fun 'Elixir.Phoenix.Endpoint.Watcher':watch/3,[<<110,111,100,101>>,[<<110,111,100,101,95,109,111,100,117,108,101,115,47,119,101,98,112,97,99,107,47,98,105,110,47,119,101,98,112,97,99,107,46,106,115>>,<<45,45,109,111,100,101>>,<<100,101,118,101,108,111,112,109,101,110,116>>,<<45,45,119,97,116,99,104,45,115,116,100,105,110>>],[{cd,<<99,58,47,85,115,101,114,115,47,74,97,109,101,115,32,67,114,111,99,107,101,114,47,100,101,118,47,80,117,122,122,108,101,83,112,97,99,101,47,112,117,122,122,108,101,115,112,97,99,101,47,97,115,115,101,116,115>>}]],{watcher_command_error,[{'Elixir.Phoenix.Endpoint.Watcher',watch,3,[{file,[108,105,98,47,112,104,111,101,110,105,120,47,101,110,100,112,111,105,110,116,47,119,97,116,99,104,101,114,46,101,120]},{line,37}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,[108,105,98,47,116,97,115,107,47,115,117,112,101,114,118,105,115,101,100,46,101,120]},{line,90}]},{proc_lib,init_p_do_apply,3,[{file,[112,114,111,99,95,108,105,98,46,101,114,108]},{line,249}]}]}]}}
2020-08-13 00:46:02.820 [error] <0.837.0> CRASH REPORT Process <0.837.0> with 0 neighbours exited with reason: watcher_command_error in 'Elixir.Phoenix.Endpoint.Watcher':watch/3 line 37
2020-08-13 00:46:02.820 [error] <0.445.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child #Ref<0.3515569155.2669936641.131262> started with 'Elixir.Phoenix.Endpoint.Watcher':start_link({node,[<<"node_modules/webpack/bin/webpack.js">>,<<"--mode">>,<<"development">>,<<"--watch-stdin">>],...}) at <0.837.0> exit with reason watcher_command_error in context child_terminated
2020-08-13 00:49:06.098 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 00:49:06.126 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 00:49:06.188 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 00:49:06.216 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 00:49:06.231 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 00:49:06.266 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 00:49:06.281 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 00:49:06.517 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 00:49:06.517 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 00:49:06.531 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 00:49:06.536 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 00:49:06.547 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 00:49:06.580 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 00:49:06.599 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 00:49:06.630 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 00:49:06.630 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 00:49:06.664 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 00:49:06.683 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 00:49:06.725 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 00:49:06.766 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 00:49:06.796 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 00:49:06.799 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 00:49:06.813 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 00:49:06.858 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 00:49:06.869 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 00:49:06.876 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 00:49:06.905 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 00:49:06.979 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 00:49:06.993 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 00:49:07.018 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 00:49:07.035 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 00:49:08.529 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 01:24:50.936 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 01:24:50.961 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 01:24:51.025 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 01:24:51.048 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 01:24:51.065 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 01:24:51.100 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 01:24:51.113 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 01:24:51.325 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 01:24:51.325 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 01:24:51.340 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 01:24:51.345 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 01:24:51.364 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 01:24:51.393 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 01:24:51.419 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 01:24:51.457 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 01:24:51.458 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 01:24:51.464 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 01:24:51.531 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 01:24:51.547 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 01:24:51.555 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 01:24:51.578 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 01:24:51.659 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 01:24:51.673 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 01:24:51.687 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 01:24:51.691 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 01:25:46.043 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 01:25:46.067 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 01:25:46.118 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 01:25:46.145 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 01:25:46.160 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 01:25:46.191 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 01:25:46.224 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 01:25:46.458 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 01:25:46.458 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 01:25:46.473 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 01:25:46.482 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 01:25:46.495 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 01:25:46.543 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 01:25:46.571 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 01:25:46.604 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 01:25:46.604 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 01:25:46.614 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 01:25:46.643 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 01:25:46.658 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 01:25:46.664 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 01:25:46.682 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 01:25:46.767 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 01:25:46.792 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 01:25:46.817 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 01:25:46.826 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 01:25:56.334 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 01:25:56.359 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 01:25:56.438 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 01:25:56.475 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 01:25:56.509 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 01:25:56.576 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 01:25:56.600 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 01:25:56.833 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 01:25:56.833 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 01:25:56.847 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 01:25:56.852 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 01:25:56.866 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 01:25:56.902 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 01:25:56.921 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 01:25:56.971 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 01:25:56.971 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 01:25:57.011 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 01:25:57.041 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 01:25:57.091 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 01:25:57.129 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 01:25:57.159 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 01:25:57.168 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 01:25:57.176 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 01:25:57.223 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 01:25:57.234 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 01:25:57.239 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 01:25:57.274 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 01:25:57.353 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 01:25:57.369 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 01:25:57.391 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 01:25:57.405 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 01:25:58.707 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 01:35:59.509 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 01:35:59.539 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 01:35:59.600 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 01:35:59.630 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 01:35:59.647 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 01:35:59.697 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 01:35:59.715 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 01:35:59.969 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 01:35:59.969 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 01:35:59.990 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 01:35:59.996 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 01:36:00.040 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 01:36:00.092 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 01:36:00.138 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 01:36:00.190 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 01:36:00.191 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 01:36:00.261 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 01:36:00.286 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 01:36:00.338 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 01:36:00.370 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 01:36:00.400 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 01:36:00.406 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 01:36:00.411 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 01:36:00.460 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 01:36:00.472 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 01:36:00.474 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 01:36:00.493 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 01:36:00.595 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 01:36:00.608 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 01:36:00.622 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 01:36:00.626 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 01:36:02.117 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 01:45:42.174 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 01:45:42.203 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 01:45:42.261 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 01:45:42.296 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 01:45:42.311 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 01:45:42.345 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 01:45:42.359 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 01:45:42.577 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 01:45:42.577 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 01:45:42.593 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 01:45:42.598 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 01:45:42.610 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 01:45:42.644 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 01:45:42.666 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 01:45:42.696 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 01:45:42.696 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 01:45:42.739 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 01:45:42.759 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 01:45:42.799 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 01:45:42.831 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 01:45:42.878 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 01:45:42.881 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 01:45:42.890 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 01:45:42.932 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 01:45:42.945 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 01:45:42.948 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 01:45:42.980 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 01:45:43.061 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 01:45:43.075 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 01:45:43.101 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 01:45:43.116 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 01:45:44.804 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 01:49:39.335 [info] <0.44.0> Application lager started on node nonode@nohost
2020-08-13 01:49:39.375 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-08-13 01:49:39.510 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-08-13 01:49:39.541 [info] <0.44.0> Application tools started on node nonode@nohost
2020-08-13 01:49:39.577 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-08-13 01:49:39.658 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-08-13 01:49:39.679 [info] <0.44.0> Application recon started on node nonode@nohost
2020-08-13 01:49:39.953 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-08-13 01:49:39.953 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-08-13 01:49:39.970 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-08-13 01:49:39.975 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-08-13 01:49:39.992 [info] <0.44.0> Application telemetry_metrics started on node nonode@nohost
2020-08-13 01:49:40.029 [info] <0.44.0> Application telemetry_poller started on node nonode@nohost
2020-08-13 01:49:40.054 [info] <0.44.0> Application jason started on node nonode@nohost
2020-08-13 01:49:40.101 [info] <0.44.0> Application comeonin started on node nonode@nohost
2020-08-13 01:49:40.101 [info] <0.44.0> Application pbkdf2_elixir started on node nonode@nohost
2020-08-13 01:49:40.148 [info] <0.44.0> Application connection started on node nonode@nohost
2020-08-13 01:49:40.176 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-08-13 01:49:40.211 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-08-13 01:49:40.264 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-08-13 01:49:40.313 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-08-13 01:49:40.316 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-08-13 01:49:40.326 [info] <0.44.0> Application phoenix_html started on node nonode@nohost
2020-08-13 01:49:40.368 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-08-13 01:49:40.380 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-08-13 01:49:40.383 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-08-13 01:49:40.413 [info] <0.44.0> Application file_system started on node nonode@nohost
2020-08-13 01:49:40.483 [info] <0.44.0> Application phoenix_live_reload started on node nonode@nohost
2020-08-13 01:49:40.498 [info] <0.44.0> Application phoenix_live_view started on node nonode@nohost
2020-08-13 01:49:40.511 [info] <0.44.0> Application phoenix_live_dashboard started on node nonode@nohost
2020-08-13 01:49:40.516 [info] <0.44.0> Application phoenix_ecto started on node nonode@nohost
2020-08-13 01:49:41.815 [info] <0.44.0> Application puzzlespace started on node nonode@nohost
2020-08-13 02:16:06.321 [error] <0.929.0> CRASH REPORT Process <0.929.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.Ecto.Query.CastError',message => <<"deps/ecto/lib/ecto/repo/queryable.ex:418: value `{:ok, %Puzzlespace.User{__meta__: #Ecto.Schema.Metadata<:loaded, \"users\">, auth_tokens: #Ecto.Association.NotLoaded<association :auth_tokens is not loaded>, entity_id: \"55e18438-e69f-4e10-a2df-bdd9dc0aadd6\", hashed_pass: \"$pbkdf2-sha512$160000$MIJbvhY4dDDfEdwurSR.mw$4juBq4nf9ejXRh4u0zEmdclRE4aFtiYOP0jkXNBJLGnMtmLHOgPQLF3LB2vlAqed1v5xQ0fJ8VVQfAABnTAoOA\"...">>,...},...},...},...},...}
2020-08-13 02:16:06.511 [error] <0.929.0> Cowboy stream 3 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.919.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.Ecto.Query.CastError',message => <<"deps/ecto/lib/ecto/repo/queryable.ex:418: value `{:ok, %Puzzlespace.User{__meta__: #Ecto.Schema.Metadata<:loaded, \"users\">, auth_tokens: #Ecto.Association.NotLoaded<association :auth_tokens is not loaded>, entity_id: \"55e18438-e69f-4e10-a2df-bdd9dc0aadd6\", hashed_pass: \"$pbkdf2-sha512$160000$MIJbvhY4dDDfEdwurSR.mw$4juBq4nf9ejXRh4u0zEmdclRE4aFtiYOP0jkXNBJLGnMtmLHOgPQLF3LB2vlAqed1v5xQ0fJ8VVQfAABnTAoOA\", ...">>,...},...},...},...}
2020-08-13 02:17:59.621 [error] <0.977.0> CRASH REPORT Process <0.977.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.Phoenix.ActionClauseError',args => [#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 134,cert => undefined,has_body => true,has_read_body => true,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"en-US,en;q=0.5">>,<<"connection">> => <<"keep-a...">>,...},...}},...},...],...},...},...},...},...}
2020-08-13 02:17:59.627 [error] <0.977.0> Cowboy stream 5 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.941.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.Phoenix.ActionClauseError',args => [#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 134,cert => undefined,has_body => true,has_read_body => true,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"en-US,en;q=0.5">>,<<"connection">> => <<"keep-ali...">>,...},...}},...},...],...},...},...},...}
